prow/plugins/approve/approvers/README.md

```diff
@@ -2,92 +2,95 @@
+3. assign each potential reviewer with a weightage by summing up weightages of all changed files in which s/he is a reviewer
 
-(4) randomly select 2 reviewers based on their weightage
+4. randomly select 2 reviewers based on their weightage
 
 ## Approval Handler and the Approved Label
 
 ### approved Label
 
-A PR cannot be merged into the Kubernetes repo without the **approved** label.  In order for the approved label to be applied, every file modified by the PR must be approved (via /approve) by an approver from the OWNERs files.  Note, this does not necessarily require multiple approvers.  The process is best illustrated in the example below.
+A PR cannot be merged into the repo without the **approved** label.  In order for the approved label to be applied, every file modified by the PR must be approved (via `/approve`) by an approver from the OWNERs files.  Note, this does not necessarily require multiple approvers.  The process is best illustrated in the example below.
 
-**Approval Selection Mechanism**
+#### Approval Selection Mechanism
 
 First, it is important to understand that ALL approvers in an OWNERS file can approve any file in that directory AND its subdirectories. Second, it is important to understand the somewhat-competing goals of the bot when selecting approvers:
 
 1. Provide a subset of approvers that can approve all files in the PR
 
 2. Provide a small subset of approvers and suggest the same reviewers as blunderbuss if possible (people can be both reviewers and approvers)
 
-3. Do not always suggest the same set of people to approver and do not consistently suggest people from the root OWNERS file
+3. Do not always suggest the same set of people to approve and do not consistently suggest people from the root OWNERS file
 
 The exact algorithm for selecting approvers is somewhat complex; it is an set cover approximation with consideration for existing assignees. To read it in depth, check out the approvers source code linked at the end of the README.  
 
-**Example**
+## Example
 
 ![Directory Structure](images/directory_structure.png)
 
 Suppose files in directories E and G are changed in a PR created by PRAuthor. Any combination of approver(s) listed below can approve the PR in order to get it merged:
 
-(1) approvers found in OWNERS files for leaf (current) directories E and G
+1. approvers found in OWNERS files for leaf (current) directories E and G
 
-(2) approvers found in OWNERS files for parent directories B and C
+2. approvers found in OWNERS files for parent directories B and C
 
-(3) approvers found in OWNERS files for root directory A
+3. approvers found in OWNERS files for root directory A
 
 Note someone can be both a reviewer found in OWNERS files for directory A and E. If s/he is selected as an approver and gives approval, it approves entire PR because s/he is also a reviewer for the root directory A.
 
-**Step 1:**
+### Step 1:
```

Comment:
It looks like the some of the paths in these examples specify `/A/B/G/OWNERS`, which doesn't exist, instead of `/A/C/G/OWNERS`.

Reply:
Every single one. Good catch!